### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by alexander871015**
Update README to describe how to start the server and extension

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,16 @@ Apache header:
...
+    1.  install bazel following the steps of 
+        https://docs.bazel.build/versions/3.2.0/install.html
+    2.  Run "bazel build //server" from the hologram_dashboard folder
+    3.  Run "bazel run //server" to start listening on port 8000
+
+## Extension
+Follow the following steps to upload the extension onto chrome
+    1.  Navigate to chrome://extensions from a chrome browser
+    2.  Enable developer's mode
+    3.  Click "load unpacked" to load "hologram_dashboard/front-end/"
```
Comment (sarahchen-google) [inline README.md:43]: Please move the instructions under hologram_dashboard/

File: hologram_dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Server 
+Follow the following steps to run the server 
+    1.  install bazel following the steps of
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:3]: Please make the instruction more concrete, for example,

1. git clone ...
2. cd ???
3. install bazel
4. ...

File: hologram_dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## Preparation
+    1.  clone the repository with "git clone
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:2]: 1. Clone the repository
    git clone https://github.com/googleinterns/sarahchen-intern-2020.git

File: hologram_dashboard/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+## Preparation
+    1.  clone the repository with "git clone 
+        https://github.com/googleinterns/sarahchen-intern-2020.git
+    2.  navigate into the "hologram_dashboard" folder with
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:4]: 2. Navigate into folder "hologram_dashboard"
    cd hologram_dashboard

File: hologram_dashboard/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+## Preparation
+    1.  clone the repository with "git clone 
+        https://github.com/googleinterns/sarahchen-intern-2020.git
+    2.  navigate into the "hologram_dashboard" folder with 
+        "cd hologram_dashboard"
+
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:6]: 3. Install bazel
    XXX

File: hologram_dashboard/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+## Server 
+Follow the following steps to run the server 
+    1.  install bazel following the steps of 
+        https://docs.bazel.build/versions/3.2.0/install.html
+    2.  Run "bazel build //server" from the hologram_dashboard folder
+    3.  Run "bazel run //server" to start listening on port 8000
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:12]: 2. Start Server
    bazel run XXX

File: hologram_dashboard/README.md (hunk 5 above)
Comment (alexander871015) [reply hologram_dashboard/README.md:6]: bazel installation is dependent on the system the user is operating on should I just assume for ubuntu?

File: hologram_dashboard/README.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+## Preparation
+    1.  clone the repository
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:2]: clone -> Clone.

File: hologram_dashboard/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+## Preparation
+    1.  clone the repository 
+        git clone https://github.com/googleinterns/sarahchen-intern-2020.git
+    2.  navigate into the "hologram_dashboard" folder
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:4]: navigate ->  Navigate

File: hologram_dashboard/README.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+        cd hologram_dashboard
+
+## Server 
+Follow the following steps to run the server 
+    1.  install bazel following the steps of
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:9]: uppercase.

File: hologram_dashboard/README.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    2.  navigate into the "hologram_dashboard" folder
+        cd hologram_dashboard
+
+## Server 
+Follow the following steps to run the server 
+    1.  install bazel following the steps of 
+        https://docs.bazel.build/versions/3.2.0/install.html
+    2.  Build the server from the hologram_dashboard folder
+        bazel build //server
+    3.  run the server to start listening on port 8000
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:13]: uppercase

File: hologram_dashboard/README.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    3.  Run the server to start listening on port 8000
+        bazel run //server
+
+## Extension
+Follow the following steps to upload the extension onto chrome
+    1.  Navigate to chrome://extensions from a chrome browser
+    2.  Enable developer's mode by clicking the developer mode option on the
+        top right
+    3.  Click "load unpacked" to load "hologram_dashboard/front-end/" then the
+        extension should show on your dashboard with a google logo
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:23]: Add some screenshot for these steps.

File: hologram_dashboard/README.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+        cd hologram_dashboard
+
+## Server 
+Follow the following steps to run the server 
+
+    1.  Install bazel by following the steps of 
+        https://docs.bazel.build/versions/3.2.0/install.html
+        for your specific system eg. ubuntu
+    2.  Build the server from the hologram_dashboard folder
+        bazel build //server
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:14]: I think it should be 

bazel build //server:server

in order to build the target binary.

File: hologram_dashboard/README.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+    3.  Run the server to start listening on port 8000
+        bazel run //server
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:16]: bazel run //server:server

File: hologram_dashboard/README.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+
+## Extension
+Follow the following steps to upload the extension onto chrome
+![Extension Screen](https://developer.chrome.com/static/images/get_started/load_extension.png)
+
+    1.  Navigate to chrome://extensions from a chrome browser
+    2.  Enable developer's mode by clicking the developer mode option on the
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:23]: I mean add screenshot like this

https://screenshot.googleplex.com/C9o8CfEfG3P

File: hologram_dashboard/README.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+        top right
+    3.  Click "load unpacked" to load "hologram_dashboard/front-end/" then the
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:25]: https://screenshot.googleplex.com/ZHXFaaQtY8W

File: hologram_dashboard/README.md [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+        extension should show on your dashboard with a google logo
+        
+## Fetcher
+Follow the following steps to start the fetching process
+
+    1.  Build the fetcher from the hologram_dashboard folder
+        bazel build //fetcher
+    2.  Run the fetcher
+        bazel run //fetcher
```
Comment (sarahchen-google) [inline hologram_dashboard/README.md:34]: Add this parter after code is ready.

File: README.md [hunk 17]
```diff
@@ -28,3 +28,16 @@ Apache header:
...
     limitations under the License.
+
+## Server 
+Follow the following steps to run the server 
+    1.  install bazel following the steps of 
+        https://docs.bazel.build/versions/3.2.0/install.html
+    2.  Run "bazel build //server" from the hologram_dashboard folder
+    3.  Run "bazel run //server" to start listening on port 8000
+
+## Extension
```
Comment (xintao1030) [inline README.md]: Good job!

File: hologram_dashboard/README.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
+## Preparation
+    1.  Clone the repository 
+        git clone https://github.com/googleinterns/sarahchen-intern-2020.git
+    2.  Navigate into the "hologram_dashboard" folder
```
Comment (xintao1030) [inline hologram_dashboard/README.md:4]: Good job

File: README.md [hunk 19]
```diff
@@ -27,4 +27,4 @@ Apache header:
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
-    limitations under the License.
```
Comment (kaihuajiang) [inline README.md:30]: Test new comments

**Pull request merged**